### DISCUSSION TITLE
Do not replace same version with suffix

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#30005] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I have yamls with versions that are based on a regex manager, but also can contain a manually added suffix. If the version (minus the suffix) is the same, I don't want it to be replaced, but currently it does. See example.

```yaml
# renovate: datasource=helm depName=mongodb repository=https://charts.bitnami.com/bitnami
version: 13.18.5-1
```

```json
{
    "customType": "regex",
    "fileMatch": ["Chart.yaml"],
    "matchStrings": ["# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( repository=(?<registryUrl>.*?))?( versioning=(?<versioning>.*?))?\\sversion: (?<currentValue>.*)(\\s|$)"]
}
```

In the above example, the current version `13.18.5-1` is replaced with the same version without the suffix `13.18.5`. In this case, the replacement should NOT happen because the actual version is unchanged. But if the version was new, like `13.19.0`, then the suffix can be removed. I tried something like: `"extractVersionTemplate": "(?<version>.*?)-\\d"`, but then the suffix was not removed.

Is it possible to do what I'm asking?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Use regex versioning with `revision`: https://docs.renovatebot.com/modules/versioning/regex/
